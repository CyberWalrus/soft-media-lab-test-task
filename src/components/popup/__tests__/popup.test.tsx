import React from 'react';
import { shallow } from 'enzyme';
import Popup from '../popup';

describe('Popup component', () => {
  it('should renders correctly', () => {
    const wrapper = shallow(<Popup>Hello</Popup>);
    expect(wrapper.isEmptyRender()).not.toEqual(true);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
