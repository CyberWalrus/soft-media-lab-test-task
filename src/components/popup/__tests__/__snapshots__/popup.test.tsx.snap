// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup component should renders correctly 1`] = `"<div class=\\"popup\\"><button class=\\"popup__button\\" type=\\"button\\"><span class=\\"popup__button-text\\">i</span></button></div>"`;
